crates/forge/tests/e2e/common/runner.rs

```diff
@@ -60,6 +66,12 @@ pub(crate) fn setup_package_with_file_patterns(
         value(get_assert_macros_version().unwrap().to_string());
     scarb_toml["target.starknet-contract"]["sierra"] = value(true);
 
+    if package_path.contains("docs/listings") {
+        scarb_toml["dev-dependencies"]["snforge_std"]
+            .as_table_mut()
+            .and_then(|snforge_std| snforge_std.remove("workspace"));
+    }
```

Comment:
Don't think we should be hardcoding some docs specific behaviors here. This could be a separate function that shares the logic with this one. 

\+ I think you wouldn't have to change the interface of current `setup_package_with_file_patterns` as well.

Reply:
I see that you created one below, but I'd try to change this so the interface of tests we already had remains unchanged.